migration-engine/core/tests/migration_tests.rs

```diff
@@ -800,14 +885,13 @@ fn adding_unique_to_an_existing_field_must_work() {
             .indices
             .iter()
             .find(|i| i.columns == vec!["field"]);
-        assert_eq!(index.is_some(), true);
+        assert!(index.is_some());
```

Comment:
I changed these because the error message is better (something like "panicked at index.is_some()" instead of "panicked at true != false").

Reply:
👍 . I am just lazy and want to type always the same macro name :-D